pkg/proxy/ipvs/proxier.go

```diff
@@ -273,6 +273,14 @@ type Proxier struct {
 	// Inject for test purpose.
 	networkInterfacer     utilproxy.NetworkInterfacer
 	gracefuldeleteManager *GracefulTerminationManager
+	// localNoEndpoints represents the number of rules that couldn't be applied due to
+	// the absence of local endpoints. It is used to publish the sync_proxy_rules_no_endpoints_total
+	// metric with the traffic_policy label set to "internal".
```

Comment:
This should say `when the traffic policy label set to "Local"`.

Reply:
I don't see this change?